doc/taoQtiTestPreviewer.yaml

```
@@ -28,6 +28,15 @@ paths:
           $ref: '#/components/responses/InitResponse'
         '400':
           $ref: '#/components/responses/BadResponse'
+
+  /taoQtiTestPreviewer/TestPreviewer/getConfiguration:
```

```suggestion
  /taoQtiTestPreviewer/TestPreviewer/configuration:
```
IMO get is redundant here, wdyt?